Source/diablo.cpp

```diff
@@ -148,6 +148,9 @@ bool was_ui_init = false;
 void StartGame(interface_mode uMsg)
 {
 	zoomflag = true;
+	if (*sgOptions.Gameplay.showItemLabels && !IsHighlightingLabelsEnabled()) {
+		ToggleItemLabelHighlight();
+	}

```

Comment:
Could we replace all occurrences of `invertHighlightToggle` with `sgOptions.Gameplay.showItemLabels`?

With this
- we don't need this code
- if right control is pressed the state would be remembered
- it's still accessible via settings for players without a keyboard

What do you think? 🙂

Reply:
It's a good idea. But I can't figure out how to actually set the sgOptions.Gameplay.showItemLabels. For ToggleItemLabelHighlight if you do ```*sgOptions.Gameplay.showItemLabels = !* sgOptions.Gameplay.showItemLabels``` it complains and I tried a bunch of different ways, but it always gives an error. I think that I need to find out how to use the ini saving function somehow in there as the way to change it but that is above my head.

Reply:
`sgOptions.Gameplay.showItemLabels.SetValue(!*sgOptions.Gameplay.showItemLabels)` should work 🙂

Reply:
That worked thanks. Has been pushed with the changes.